extensions/google-java/src/main/kotlin/build/buf/connect/extensions/GoogleJavaJSONAdapter.kt

```diff
@@ -12,10 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package build.buf.connect.extensions
+package com.connectrpc.extensions
```

Comment:
This file should move to the new package too under src/main/kotlin/com/connectrpc/extensions.